src/mtev_reverse_socket.c

```diff
@@ -797,12 +799,15 @@ mtev_reverse_socket_handler(eventer_t e, int mask, void *closure,
         channel_closure_t *cct;
         cct = malloc(sizeof(*cct));
         cct->channel_id = rc->data.incoming_inflight.channel_id;
+        // Anchor 'rc' to the life cycle of 'newe'
+        mtev_reverse_socket_ref(&rc->refcnt);
```

Comment:
Am I missing something or is this meant to be passing `rc` rather than `&rc->refcnt`?

Reply:
yes, copy and paste error